src/argdescriber.h

```diff
@@ -13,7 +13,7 @@ namespace argparse {
 
 class Option;
 class Command;
-class ParserDefinition;
+struct ParserDefinition;
```

Comment:
I think it would be better if we changed ParserDefinition to class and make its members public.